configure.ac

```diff
@@ -156,6 +156,7 @@ if test "x$FHS_ENABLED" = xyes ; then
 	AC_SUBST(domserver_rundir,        $localstatedir/run/${PACKAGE_TARNAME})
 	AC_SUBST(domserver_tmpdir,        /tmp)
 	AC_SUBST(domserver_exampleprobdir,$datadir/${PACKAGE_TARNAME}/example_problems)
+	AC_SUBST(domserver_databasedumpdir,$datadir/${PACKAGE_TARNAME}/db-dumps)
```

Comment:
I don't think this should be datadir as those may not be writable by the user, but probably something under `/var`, maybe `/var/lib` or `/var/tmp`, but please check https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
Note that `$localstatedir` is `/var` so something else needs to be added here: stuff shouldn't just go under `/var/domjudge`. Probably `/var/lib/domjudge` is ok. WDYT @thijskh ?

Reply:
I've started the discussion in maintainer for this (as there are more directories which should be discussed) for now I've used /var/lib to stay in line with judgings.